packages/@sanity/base/src/components/validation/listItem.tsx

```diff
@@ -0,0 +1,75 @@
+import {Marker, Path} from '@sanity/types'
+import React, {useCallback, useMemo} from 'react'
+import {WarningOutlineIcon, ErrorOutlineIcon} from '@sanity/icons'
+import {Box, Text, MenuItem, Stack, Flex, Theme, CardTone} from '@sanity/ui'
+import styled from 'styled-components'
+
+interface ValidationListItemProps {
+  marker: Marker
+  onClick?: (path?: Path) => void
+  path: string
+  // showLink?: boolean
+  truncate?: boolean
+}
+
+const StyledText = styled(Text)`
+  white-space: initial;
+`
+
+const StyledMenuItem = styled(MenuItem)`
+  ${({theme, tone}: {theme: Theme; tone: CardTone}) => `
+  
+  &:not([hidden]) {
+    --card-bg-color: ${theme.sanity.color.card.enabled.bg};
+    --card-fg-color: ${theme.sanity.color.card.enabled.fg};
+    --card-muted-fg-color: ${theme.sanity.color.card.enabled.fg};
+  }
+
+  &[data-as='button']:not(:disabled):focus  {
+    --card-bg-color: ${theme.sanity.color.muted[tone].hovered.bg};
+    --card-fg-color: ${theme.sanity.color.muted[tone].hovered.fg};
+    --card-muted-fg-color: ${theme.sanity.color.muted[tone].hovered.fg};
+  }
+`}
+`
+
+export function ListItem(props: ValidationListItemProps) {
+  const {marker, onClick, path, truncate} = props
+
+  const tone = useMemo(() => (marker.level === 'warning' ? 'caution' : 'critical'), [marker.level])
```

Comment:
This shouldn't need any memoization:
```suggestion
  const tone = marker.level === 'warning' ? 'caution' : 'critical'
```

Reply:
ok!